runtime/caml/misc.h

```diff
@@ -688,6 +688,9 @@ void caml_gc_log (const char *, ...)
 extern uintnat caml_runtime_warnings;
 CAMLextern int caml_runtime_warnings_active(void);
 
+extern intnat caml_float_array_alloc_warning;
+int caml_float_array_alloc_warning_active(void);
```

Comment:
I guess `CAMLextern` should be added here? (By analogy with `caml_runtime_warnings_active`).
```suggestion
CAMLextern int caml_float_array_alloc_warning_active(void);
```